src/app.js

```diff
@@ -1,9 +1,9 @@
 const express = require('express');
-const api = require('./api');
+const userRoutes = require('./routes/userRoutes');
 
 const app = express();
 
 app.use(express.json());
-app.use('/api/v1', api);
+app.use('/api/v1/users', userRoutes);
 
```

Comment:
I would create a  router file to remove all of the `app.use(router)` from here, something like this : 

```js
const express = require('express');
const router = express.Router();

const firstRouter = require('./firstRouter');
const secondRouter = require('./secondRouter');

router.use(firstRouter);
router.use(secondRouter);

module.exports = router;

```

Then call this form `app.js`